var/spack/repos/builtin/packages/superlu/package.py

```diff
@@ -3,6 +3,8 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import os
```

Comment:
You can remove this after switching to `join_path` for consistency elsewhere in the package.